NeoMathEngine/src/CPU/CpuMathEngineDnnConv.cpp

```diff
@@ -431,14 +408,10 @@ void CCpuMathEngine::BlobConvolution( const CConvolutionDesc& convDesc, const CC
 		case CA_1:
 		case CA_2:
 		{
-			const int algo0ThreadCount = IsOmpRelevant( desc.Result.ObjectCount() * desc.Result.Width() * desc.Result.Height(),
-				static_cast<int64_t>( desc.Result.BlobSize() ) * desc.Filter.ObjectSize() ) ? threadCount : 1;
+			const int64_t algo1DataSize = static_cast<int64_t>( desc.Result.Width() )
+				* desc.Result.Height() * desc.Filter.ObjectSize() + desc.Result.ObjectSize();
 
-			const int algo1ThreadCount = IsOmpRelevant( desc.Result.ObjectCount() * desc.Result.Width(),
-				static_cast<int64_t>( desc.Result.BlobSize() ) * desc.Filter.ObjectSize() ) ? threadCount : 1;
-			const int64_t algo1DataSize = static_cast<int64_t>( desc.Result.Width() ) * desc.Result.Height() * desc.Filter.ObjectSize() + desc.Result.ObjectSize();
-
-			if( std::min( desc.Result.ObjectCount(), algo1ThreadCount ) * algo1DataSize <= algo0ThreadCount * BlobConvolutionCacheSize ) {
+			if( desc.Result.ObjectCount() * algo1DataSize <= BlobConvolutionCacheSize ) {
```

Comment:
`std::min( desc.Result.ObjectCount(), algo1ThreadCount )` is `1`, not `desc.Result.ObjectCount()`

Reply:
fixed